.github/workflows/release.yml

```diff
@@ -0,0 +1,129 @@
+            git fetch --tags -q 2>/dev/null
+            tag_exists=$( git tag --list "$version" )
+          
+            if [[ -z "$tag_exists" ]]
+            then
+              echo "A tag for $version does not exist."
+              exit 1
+            fi
+            
+            echo ::set-output name=update::true
+          else
+            version=$( cat core/Version.php | grep -oP "VERSION = '\K([^\']+)" )
+            echo "Version to build: '$version'"
+          
+            git fetch --tags -q 2>/dev/null
+            tag_exists=$( git tag --list "$version" )
+          
+            if [[ -n "$tag_exists" ]]
+            then
+              echo "A tag for $tag_exists already exists."
+              exit 1
+            fi
+          
+            echo "Creating a tag for $version"
+          
+            git tag $version
+            git push origin tags/$version
+            
+            echo ::set-output name=update::false
+          fi
+          
+          if [[ "$version" =~ "-" ]]
+          then
+            echo ::set-output name=prerelease::true
+            body="## Matomo ${version} (Pre-release)
+            
+            We recommend to read [this FAQ](http://matomo.org/faq/how-to-update/faq_159/) before using a pre-release in a production environment.
+            
+            Please use the attached archives for installing or updating Matomo.
+            The source code download is only meant for developers and will require extra work to install it. 
+             - Latest stable production release can be found at https://matomo.org/download/ ([learn more](https://matomo.org/docs/installation/)) (recommended)
+             - Beta and Release Candidates releases can be found at https://builds.matomo.org/ ([learn more](http://matomo.org/faq/how-to-update/faq_159/))"
+          else
+            echo ::set-output name=prerelease::false
+            body="## [Matomo ${version} Changelog](https://matomo.org/changelog/matomo-${version//./-}/)
+            
+            Please use the attached archives for installing or updating Matomo.
+            The source code download is only meant for developers and will require extra work to install it. 
+             - Latest stable production release can be found at https://matomo.org/download/ ([learn more](https://matomo.org/docs/installation/)) (recommended)
+             - Beta and Release Candidates releases can be found at https://builds.matomo.org/ ([learn more](http://matomo.org/faq/how-to-update/faq_159/))"
+          fi
+
+          body="${body//'%'/'%25'}"
+          body="${body//$'\n'/'%0A'}"
+          body="${body//$'\r'/'%0D'}"
+          echo ::set-output name=version::$version
+          echo ::set-output name=body::$body
+
+          git clone https://github.com/matomo-org/matomo-package.git matomo-package
+          cd matomo-package
+          ./scripts/build-package.sh $version
+      shell: bash 
+    - uses: ncipollo/release-action@v1
```

Comment:
can we use https://github.com/github/hub maybe instead of this action?

Like

`hub release create --message $VERSION --message "If…`

Not sure if we need "allowUpdates etc"

Otherwise should pin the action to a specific commit.